pkg/result/filter_test.go

```diff
@@ -713,6 +713,65 @@ func TestFilter(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "ignore policy directory",
+			args: args{
+				report: types.Report{
+					Results: types.Results{
+						{
+							Vulnerabilities: []types.DetectedVulnerability{
+								vuln1,
+								vuln2, // filtered by severity
+								vuln3, // ignored by policy
+							},
+							Misconfigurations: []types.DetectedMisconfiguration{
+								misconf1,
+								misconf2, // filtered by status
+								misconf3, // ignored by policy
+							},
```

Comment:
nit: 
In this test we check rego policy.
So I think we don't need to include `vuln2` and `misconf2`.

Reply:
Makes sense. Removed.